### PR TITLE
feat(cmf): add cmf.collection.includes expression

### PR DESCRIPTION
**Pull request by romainseb**
**What is the problem this PR is trying to solve?**
Sometimes, information is not accessible with a getIn, as when we have a list in the store.
**What is the chosen solution to this problem?**
To provide access to this, i set here 2 new accessors to know if a value is present in a list in the store, theirs pretty names are :
- cmf.collections.includes
- cmf.components.includes

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [x] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/cmf/src/expressions/getIn.includes.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import Immutable from 'immutable';
+import curry from 'lodash/curry';
+
+function getInIncludes(statePath, { context }, immutablePath, value) {
+	return _get(context.store.getState(), statePath, new Immutable.Map())
+		.getIn(immutablePath.split('.'), new Immutable.List())
+		.includes(value);
+}
+
+export default curry(getInIncludes);
```
Comment (jlame) [inline packages/cmf/src/expressions/getIn.includes.js]: Maybe use a closure here ? which i find more readable and also the function args after statePath don't seem easy to configure in advance as it is linked with cmf context. Not a issue if we stay like this.

File: packages/cmf/__tests__/expressions/index.test.js [hunk 2]
```diff
@@ -19,18 +21,61 @@ describe('expressions', () => {
...
 			});
 			context.store.getState = () => state;
-			expect(expressions['cmf.collections.get']({ context }, 'article.title', 'no title'))
-				.toBe('my title');
+			expect(expressions['cmf.collections.get']({ context }, 'article.title', 'no title')).toBe(
+				'my title',
+			);
 		});
-		it('should return default value if collection doesn\'t exists', () => {
+		it("should return default value if collection doesn't exists", () => {
```
Comment (jmfrancois) [inline packages/cmf/__tests__/expressions/index.test.js]: simple quote please

File: packages/cmf/src/expressions/getIn.includes.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import _get from 'lodash/get';
+import Immutable from 'immutable';
+import curry from 'lodash/curry';
+
+function getInIncludes(statePath, { context }, immutablePath, value) {
+	return _get(context.store.getState(), statePath, new Immutable.Map())
+		.getIn(immutablePath.split('.'), new Immutable.List())
+		.includes(value);
```
Comment (jmfrancois) [inline packages/cmf/src/expressions/getIn.includes.js]: hasIn hasn't done the job ?

File: packages/cmf/__tests__/expressions/index.test.js (hunk 2 above)
Comment (romainseb) [reply packages/cmf/__tests__/expressions/index.test.js]: Yep, it's prettier

**Pull request merged**